packages/components/src/components/gridbar/QueryGridBar.tsx

```diff
@@ -98,6 +98,10 @@ export class QueryGridBar extends React.PureComponent<QueryGridBarProps, any> {
                     {chart}
                 </div>
             );
+        } else {
+            leftContent = (
+                <div className="col-md-6 col-sm-6 col-xs-12" />
```

Comment:
Why are we always adding this now?

Reply:
Because if there is no left content column, then the right content column (size 6) becomes the left column

Reply:
If we always need it, but sometimes override it, then let's just make it the default when we define `let leftContent` above, then we don't need an indented else block here.

Reply:
Good call